neuropsychopharmacology.csl

```diff
@@ -1,128 +1,175 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
     <title>Neuropsychopharmacology</title>
+    <title-short>NPP</title-short>
     <id>http://www.zotero.org/styles/neuropsychopharmacology</id>
     <link href="http://www.zotero.org/styles/neuropsychopharmacology" rel="self"/>
-    <link href="http://www.nature.com/npp/author_instructions.html" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/stem-cells-translational-medicine" rel="self"/>
+    <link href="http://mts-npp.nature.com/cgi-bin/main.plex?form_type=display_auth_instructions" rel="documentation"/>
     <author>
-      <name>Peter Hedlund</name>
-      <email>peter.hedlund@me.com</email>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
```

Comment:
@POBrien333, did you remove the original author on purpose? (I assume so, since it looks like you basically replaced all the style code)

Reply:
This is a completely new style as if it hadn't been on the repo before.